content/docs/standard/table-schema.md

```diff
@@ -266,6 +266,44 @@ If the value of the `foreignKey.reference.resource` property is an empty string
 Data consumer MUST support the `foreignKey.fields` and `foreignKey.reference.fields` properties in a form of a single string e.g. `"fields": "a"` which was a part of the `v1.0` of the specification.
 :::
 
+#### `name`
+
+A simple name or identifier for the schema (cf. [Data Package](https://datapackage.org/standard/data-package/#name)).
+
+#### `title`
+       
+A string providing a title or one sentence description for the schema.
+
+#### `description`
+       
+A description of the schema (cf. [Data Package](https://datapackage.org/standard/data-package/#description)).
+
+#### `homepage`
+       
+A URL for the home on the web that is related to the schema.
+
+#### `keywords`
+
+An array of string keywords to assist users searching for the schema in catalogs.
```

Comment:
@pierrecamilleri would you argue this inherits from Data Package or not? If so, please change to:

```suggestion
An array of string keywords to assist users searching for the schema in catalogs. If not specified, the schema inherits from the Data Package if distributed in a Data Package descriptor.
```